index.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en-gb">
+<head>
+	<title>portfolio</title>
+	<link href="styles.css" type="text/css" rel="stylesheet" />
+</head>
+
+<body> 
+	
+	<nav>
+		<section>
+			<h1> Emily Clarkson</h1>
+			
+			<div class="page-nav">
+				<a class="about-link" href="#about">about</a>
+				<a class="work-link" href="#portfolio">work</a>
+				<a class="contact-link" href="#footer">contact</a>
+			</div>
+			
+			<div>
+				<a class="cv" href="#">download CV</a>
+			</div>
+				
+		</section>
+		
+	</nav>
+	
+	<header>
+		<section class="justify-content">
+  		   <div class="header-img">
+  				<img src= "file:///Users/emilyclarkson/Desktop/lettering-free/lettering-free/1080x1080/1_0037-new.png">
```

Comment:
Because of the way you have linked these images, I can't access them. Rather than linking to them on your desktop, what you should do instead is actually move the image files themselves into your repository (so in the same folder as this file) and then you can add them into your repository.

That way, whenever someone clones your repository, they have all the images needed to hand to display everything. 

If you moved this file `1_0037-new.png` into your project folder, the img tag would become `<img src="1_0037-new.png">

Reply:
Also don't forget to add an alt text attribute to all of your img tags